backend/api/ceph/nfs.py

```diff
@@ -0,0 +1,183 @@
+# bubbles - a simplified management UI for Ceph
+# Copyright (C) 2021 SUSE, LLC
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+from typing import Callable, List
+
+from fastapi import APIRouter, Depends, HTTPException, Request, status
+
+from bubbles.bubbles import Bubbles
+from bubbles.backend.api import jwt_auth_scheme
+from bubbles.backend.controllers.ceph.nfs import (
+    Error,
+    NotFound,
+)
+from bubbles.backend.models.ceph.nfs import (
+    CephFSExportRequest,
+    NFSExportModel,
+    NFSServiceModel,
+    NFSServiceRequest,
+)
+
+router = APIRouter(prefix="/ceph/nfs", tags=["ceph"])
+
+
+@router.put(
+    "/service/{name}",
+    name="create an nfs service",
```

Comment:
While I'd agree with the `nfs` to `NFS` change, the `an` is actually the right word because `NFS` is read as `ENE-EFE-ESS`, which, phonetically, starts with a vowel -- hence the `an` instead of the `a`.